debian/gtk-3-examples.install

```diff
@@ -1,3 +1,4 @@
+usr/bin/gtk-encode-symbolic-svg
```

Comment:
So we stuck these in the wrong package long ago... I remember thinking that moving them would cause tricky upgrade paths (you'd need Conflicts old libgtk-3-bin or something like that?).

Maybe my reasoning was bad and if the upgrade paths have been tested and work, I think its fine to move this into the proper package.